JCN-169-event-emitter (#1)
janis-commerce/event-emitter: README.md

[![Build Status](https://travis-ci.org/janis-commerce/event-emitter.svg?branch=master)](https://travis-ci.org/janis-commerce/event-emitter)
[![Coverage Status](https://coveralls.io/repos/github/janis-commerce/event-emitter/badge.svg?branch=master)](https://coveralls.io/github/janis-commerce/event-emitter?branch=master)

A package for emit events to janis-events

## Installation
```sh
npm install @janiscommerce/event-emitter
```

## Configuration
### ENV variables

> `JANIS_SERVICE_NAME` (required): The name of the service that will emit the event.

Reviewer: Debería decir que tiene que ir en el `serverless.yml` en `environment`

Reviewer 2: No estoy de acuerdo con esto... es un package que funciona en cualquier lado.
En otro documento se debe explicar, pero no en este.

Reviewer: Entonces talvés estaría bueno que no solo se tomará de las variables de ambiente sino que se pudiera ingresar de otro modo (via parámetros por ejemplo) si usó este paquete desde otro lado. Talvés tomando como default la de Ambiente.

Reviewer 2: La variable de entorno esta bien. Lo que no me parece bien era escribir exclusivo para `serverless.yml` :+1:

[![Build Status](https://travis-ci.org/janis-commerce/event-emitter.svg?branch=master)](https://travis-ci.org/janis-commerce/event-emitter)
[![Coverage Status](https://coveralls.io/repos/github/janis-commerce/event-emitter/badge.svg?branch=master)](https://coveralls.io/github/janis-commerce/event-emitter?branch=master)

A package for emit events to janis-events

## Installation
```sh
npm install @janiscommerce/event-emitter
```

## Configuration
### ENV variables
`JANIS_SERVICE_NAME` (required): The name of the service that will emit the event.

### Important
This package uses [@janiscommerce/microservice-call](https://www.npmjs.com/package/@janiscommerce/microservice-call), it's strongly recommended read this package docs before continue.

## API

### **`emit(event)`**

Emits an event to janis-events

> Returns `true` if the operation is successful or `false` if not.

Reviewer 2: ```suggestion
Returns `true` if the operation is successful, `false` otherwise.
```

#### Event parameter

> The event parameter is an `[Object]` and have the next struct:

Reviewer 2: ```suggestion
The event parameter is an `[Object]` and have the following structure:
```
